MapleServer2/PacketHandlers/Game/GuildHandler.cs

```diff
@@ -93,25 +117,27 @@ private void HandleCreate(GameSession session, PacketReader packet)
         {
             string guildName = packet.ReadUnicodeString();
 
-            if (session.Player.Wallet.Meso.Modify(-2000))
+            if (session.Player.Wallet.Meso.Amount < 2000)
```

Comment:
This is the wrong way of using function Modify, it returns an boolean, if the character doesn't have enough money it returns false and if it does have enough money the function changes the amount and returns true.

Reply:
What was there previously did not work. It would take your mesos and tell you you don't have enough mesos.

Reply:
It's because when I made the Wallet class I didn't change anything else outside the currency feat, so it's my bad, the IF should be inverted then it will work without a problem. Sorry!